scripts/convert_domains_and_systems_using_opslevel_cli/opslevel_cli_commands.md

```diff
@@ -0,0 +1,56 @@
+# Convert Domains and Systems using the opslevel-cli
+
+The opslevel-cli commands and steps below can be used to convert domains into systems, and systems into services.
+
+Notes on converting domains into systems:
+* Domain names will be preserved
+* Owner will be preserved
+* Description will be preserved
+
+Notes on converting Systems into services:
+* System names will be prefixed with "SEARCH-"
+* Owner will be preserved
+* Description will be preserved
+* Link to new system (previously domain) will be preserved
+
+# Requirements
+
+* opslevel-cli installed https://github.com/OpsLevel/cli
+* Ensure that the domains and systems only have 1 alias defined (the alias should be indicated as locked in the UI)
+
+# Steps and commands
+
+1. Backup systems and domains
+
+```
+opslevel list systems -o json > systems_<filename>.json
+opslevel list domains -o json > domains_<filename>.json
```

Comment:
Might want to call out that `<filename>` is something that should be replaced to ensure uniqueness - otherwise whats the point could just use `systems.json` or `existing_systems.json`

Reply:
@rocktavious changes made based on suggestions!